src/mds/MDSMap.cc

```diff
@@ -583,7 +586,7 @@ void MDSMap::decode(bufferlist::iterator& p)
   std::map<mds_rank_t,int32_t> inc;  // Legacy field, parse and drop
 
   cached_up_features = 0;
-  DECODE_START_LEGACY_COMPAT_LEN_16(5, 4, 4, p);
+  DECODE_START_LEGACY_COMPAT_LEN_16(6, 4, 4, p);
```

Comment:
Leave this alone, and when you read out 'banalcer' at the very end make it conditional on ev>=11